.github/dependabot.yml

```diff
@@ -16,6 +16,20 @@ updates:
       interval: "weekly"
     open-pull-requests-limit: 3
     labels: ["component/dependencies"]
+    ignore:
+      - # Needs to updated along with NodeJS version
+        dependency-name: "@types/node"
+        versions: [ ">16" ]
+      - # Blocked by nuxt
+        dependency-name: "vue"
```

Comment:
I think we also need to add `vue-template-compiler`

Reply:
`vue-template-compiler` is already known to typescript to be tied to `vue`, so it shouldn't matter here.